truffle-config.base.js

```diff
@@ -20,18 +20,16 @@
 require("dotenv").config();
 
 const HDWalletProvider = require("@truffle/hdwallet-provider");
-const DeployManager = require("./utils/deploy-manager.js");
+// const deployManager = require("./utils/deploy-manager.js");
 
 const _gasPrice = process.env.DEPLOYER_GAS_PRICE || 20000000000;
 const _gasLimit = 6000000;
 
-async function getKeys() {
-  const accounts = await web3.eth.getAccounts();
-  const deploymentAccount = accounts[0];
-  const manager = new DeployManager(deploymentAccount);
-  await manager.setup();
-  const { pkey, infuraKey } = manager;
-  return (pkey, infuraKey);
+function getKeys() {
+  // NOTE: While https://github.com/trufflesuite/truffle/issues/1054 is implemented we are using a temporary fix
```

Comment:
FYI we used to have a custom `AWSWalletProvider` to solve that issue https://github.com/argentlabs/solidity/blob/dev/truffle-awswallet-provider/index.js

Reply:
I'll add a card to investigate that if the truffle fix is delayed